app/assets/javascripts/registration.js

```diff
@@ -0,0 +1,47 @@
+$( document ).ready(function() {
+	$('#register-send').on('click', function(){
+		$('#regForm').validate
```

Comment:
take a desition and follow just one convention, use always `kebab-case` or `camelCase`

Reply:
https://adrianalonso.es/cajon-desatre/convencion-de-nombres-desde-el-camelcase-hasta-el-kebab-case/